src/layers/index.d.ts

```diff
@@ -79,17 +74,16 @@ export class Layer {
   _oldDataUpdateTriggers: any;
   hasAllColumns(): boolean;
   updateLayerConfig(p: Partial<LayerConfig>): Layer;
-  updateLayerDomain(datasets: Datasets, filter?: Filter): Layer;
```

Comment:
why is this type def deleted? base-layer still has `updateLayerDomain` method

Reply:
not sure i deleted this one, but i will revert it

Reply:
probably some rebase conflicts
same for 
```
isLayerHovered(objectInfo: any): boolean;
  hasHoveredObject(objectInfo: any): any | null;
  getHoverData(object: any, allData?: Dataset['allData'], fields?: Dataset['fields']): any;
```